src/number-value.js

```diff
@@ -12,27 +12,28 @@
 //     See the License for the specific language governing permissions and
 // limitations under the License.
 
-(function(shared, scope) {
+(function(shared, scope, testing) {
```

Comment:
Since these changes are unrelated, it's good practice to put them in a different PR.